Client/Frontend/Browser/Playlist/VideoPlayer/UI/VideoPlayerTrackbar.swift

```diff
@@ -234,8 +246,34 @@ class VideoTrackerBar: UIView {
     
     private func timeToString(_ time: CMTime) -> String {
```

Comment:
Since we got a bug let's add unit test for this method to catch all tricky cases